static/app/views/dashboards/widgetBuilder/utils.tsx

```diff
@@ -59,7 +59,6 @@ export const displayTypes = {
   [DisplayType.BAR]: t('Bar Chart'),
   [DisplayType.LINE]: t('Line Chart'),
   [DisplayType.TABLE]: t('Table'),
-  [DisplayType.WORLD_MAP]: t('World Map'),
```

Comment:
Is this needed at all for existing world map components loading in or is the table replacing that widget display type?

Reply:
Yeah this shouldn't affect existing world maps. You just can't add a new one.